lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
SO in this file the 'use strict' should be around like 11 inside the categoryNameConverter module function.